docker/freebsd-install.sh

```diff
@@ -6,35 +6,138 @@ set -euo pipefail
 # shellcheck disable=SC1091
 . freebsd-common.sh
 
-export PACKAGESITE=/opt/freebsd-packagesite/packagesite.yaml
-export PKG_SOURCE="https://pkg.freebsd.org/FreeBSD:${BSD_MAJOR}:${BSD_ARCH}/quarterly"
-export TARGET="${ARCH}-unknown-freebsd${BSD_MAJOR}"
+# list of SRV records to query if the default mirror fails
```

Comment:
Everything has been prefixed with `freebsd` since these could be included in user scripts.